### DISCUSSION TITLE
Combining Ambient RNA Methods

### DISCUSSION THREAD

**Discussion by Reyhaneh92** [campbio/decontX#28] [General]
Thank you for the helpful package. I'm curious about the feasibility of integrating multiple ambient RNA analysis methods simultaneously. Specifically, I have observed improved results when using SoupX in conjunction with DecontX. Is it advisable to combine these methods, and if so, are there any specific considerations or best practices to keep in mind during the integration process?

**Comment by joshua-d-campbell** [comment]
Hello @Reyhaneh92, thanks for using our tool. We have not tested it in combination with SoupX. There are different assumptions and techniques used by each algorithm so I would be hesitant to combine them. decontX can be run a few different ways as well (i.e. with and without the raw background matrix). It would be hard for me to give you a definitive answer without knowing more about your data and the performance of each tool.  If you can provide details or example, that would be helpful.
